po/it.po

```diff
@@ -342,21 +364,21 @@ msgstr[1] "secondi"
 
 #: src/pt-preferences.c:258
 msgid "Deleting configuration"
-msgstr ""
+msgstr "Eliminazione della configurazione"
 
 #: src/pt-preferences.c:259
 #, c-format
 msgid "Do you really want to delete “%s”?"
-msgstr ""
+msgstr "Vuoi veramente cancellare “%s”?"
```

Comment:
Use \\" as quotation marks, like the Italian GNOME translation team: that would be "Vuoi veramente cancellare \\"%s\\"?"